apps/daimo-mobile/package.json

```diff
@@ -18,7 +21,9 @@
     "lint:style": "eslint . --max-warnings=0",
     "fix:deps": "expo install --fix",
     "fix:style": "eslint --fix .",
-    "eas-build-post-install": "cd ../.. && npm run build"
+    "eas-build-post-install": "cd ../.. && npm run build",
+    "android": "expo run:android",
+    "ios": "expo run:ios"
```

Comment:
can delete `run:...` targets, given that expo likes to auto-add `npm run android` / `npm run ios`